tests/step_defs/ui/test_ui_admin.py

```diff
@@ -17,123 +17,133 @@
-
-
 @when("the user navigates to the home page")
-def ui_admin_home_access(browser):
+def ui_admin_navigates_to_home(browser):
     url = "{}/".format(portal_url)
     browser.visit(url)
 
 
-@when("the user navigates to the admin page")
-def ui_admin_access(browser):
-    url = "{}/admin".format(portal_url)
-    browser.visit(url)
+@then("the text Administration is shown")
+def ui_admin_administration_present(browser):
+    h1_tags = browser.find_by_tag("h1")  # Avoid finding the one in dropdown
+    found = any("administration" in h1.text.lower() for h1 in h1_tags)
+    assert found
 
 
-@when(parsers.parse("the user input banner text with message {message}"))
-def ui_admin_input_banner_msg(browser, message):
-    browser.fill("banner", message)
+@then("the Administration option is available in the menu dropdown")
+def ui_admin_administration_dropdown_present(browser):
+    xpath = "//a[@class='dropdown-item' and @href='/admin/']"
+    found = browser.is_element_present_by_xpath(xpath)
+    assert found
 
 
-@when(parsers.parse("the user {action} the checkbox to mark the banner as important"))
-def ui_admin_control_importance(browser, action):
-    if action == "checks":
-        browser.check("importance")
-    elif action == "unchecks":
-        browser.uncheck("importance")
-    else:
-        raise ValueError("Unsupported action.")
+@then("the Maintenance Banner feature is visible")
+def ui_admin_banner_option_present(browser):
+    assert browser.is_text_present("Maintenance Banner")
+    assert browser.find_by_id("admin-banner-message").visible
+    assert browser.find_by_id("admin-banner-importance").visible
+    assert browser.find_by_css("button[id='admin-set-banner']").visible
+    assert browser.find_by_css("button[id='admin-remove-banner']").visible
 
 
-@when(parsers.parse("the user click button {button}"))
-def ui_admin_click_button(browser, button):
-    browser.find_by_name(button).first.click()
+@then("the Portal Theme feature is visible")
+def ui_admin_theme_option_present(browser):
+    assert browser.is_text_present("Portal Theme")
+    assert browser.find_by_id("admin-theme-selection").visible
 
 
-@when(parsers.parse("the user change portal theme to {theme}"))
-def ui_admin_change_theme(browser, theme):
-    browser.find_by_id("theme").first.select(theme)
+@then("the Publication Terms feature is visible")
+def ui_admin_pub_terms_option_present(browser):
+    assert browser.is_text_present("Publication Terms")
+    assert browser.find_by_id("admin-publication-terms").visible
 
 
-@then("the text Administration is shown")
-def ui_admin_administration_present(browser):
-    h1_tags = browser.find_by_tag("h1")  # Avoid finding the one in dropdown
-    found = any("administration" in h1.text.lower() for h1 in h1_tags)
-    assert found
+@when("the user navigates to the admin page")
+def ui_admin_navigates_to_admin(browser):
+    url = "{}/admin".format(portal_url)
+    browser.visit(url)
```

Comment:
It would be a bit cleaner to remove this and instead use this line in the .feature file: "When module "admin" is shown"

Reply:
Hi Claire, on a second thought this original suggestion is ideal, since it is consistent with other modules. So I updated codes per this suggestion.